pkg/chains/formats/slsa/extract/extract_test.go

```diff
@@ -144,23 +144,22 @@ func createTaskRunObjectWithResults(results map[string]string) objects.TektonObj
 }
 
 func createPipelineRunObjectWithResults(results map[string]string) objects.TektonObject {
-	prResults := []v1beta1.PipelineRunResult{}
+	pro := objects.NewPipelineRunObject(nil)
 	prefix := 0
 	for url, digest := range results {
-		prResults = append(prResults,
-			v1beta1.PipelineRunResult{Name: fmt.Sprintf("%v_IMAGE_DIGEST", prefix), Value: *v1beta1.NewStructuredValues(digest)},
-			v1beta1.PipelineRunResult{Name: fmt.Sprintf("%v_IMAGE_URL", prefix), Value: *v1beta1.NewStructuredValues(url)},
-		)
+		tr := &v1beta1.TaskRun{
+			Status: v1beta1.TaskRunStatus{
+				TaskRunStatusFields: v1beta1.TaskRunStatusFields{
+					TaskRunResults: []v1beta1.TaskRunResult{
+						{Name: fmt.Sprintf("%v_IMAGE_DIGEST", prefix), Value: *v1beta1.NewStructuredValues(digest)},
+						{Name: fmt.Sprintf("%v_IMAGE_URL", prefix), Value: *v1beta1.NewStructuredValues(url)},
+					},
+				},
+			},
+		}
+		pro.AppendTaskRun(tr)
 		prefix++
 	}
 
-	return objects.NewPipelineRunObject(
```

Comment:
Let's please not drop support for PipelineRun results.

Reply:
@lcarva Done!